### PR TITLE
Update troubleshooting documentation

### PR DESCRIPTION
**Pull request by fiver-watson**
This pull request completely rewrites the Admin manual's "Troubleshooting" page, pulling from the old Qubit wiki and various user forum threads to compile useful information for developers and system administrators trying to resolve errors and other issues in AtoM. It includes 3 commits. The main rewrite is in af92438f7b952cb16be7293ffb32d9454bfe8cd0. The other commits fix related commands in other Admin manual docs, and add a bunch of ref links from other relevant documentation back to the new troubleshooting documentation.

### CODE REVIEW

File: admin-manual/maintenance/troubleshooting.rst [hunk 1]
```diff
@@ -4,37 +4,1539 @@
...
+
+
+.. _troubleshooting-restart-job-scheduler:
+
+Restarting the Job scheduler
+----------------------------
+
+**What**
+
+AtoM relies a job scheduler in order to execute certain long-running tasks
```
Comment (sallain) [inline admin-manual/maintenance/troubleshooting.rst]: Missing word here - "Atom relies **on** a job scheduler" :)

**Review by sallain — APPROVED** [review]
Looks good! I didn't read everything in detail since I'm sure that they're mostly canned responses you've provided several times, but the overall format is very useful.

**Comment by fiver-watson** [comment]
Great, thanks! I fixed that typo in the Job scheduler section, and a few others that @sevein found.

**Review by sevein — APPROVED** [review]
LGTM

**Review by jraddaoui — APPROVED** [review]
Really nice!

**Pull request merged**